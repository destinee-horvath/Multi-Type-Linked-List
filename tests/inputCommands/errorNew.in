new
NeW
NEW 
NEW
NEW5
NEW -3
NEW abc
