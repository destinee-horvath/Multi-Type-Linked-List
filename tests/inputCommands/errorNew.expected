INVALID COMMAND: INPUT
INVALID COMMAND: INPUT
INVALID COMMAND: NEW
INVALID COMMAND: NEW
INVALID COMMAND: NEW
INVALID COMMAND: NEW
INVALID COMMAND: NEW
